client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEditForm.tsx

```diff
@@ -89,13 +97,16 @@ export const ResponseLineEditForm = ({
       Left={
         <>
           <Typography variant="body1" fontWeight="bold">
-            {t('heading.stock-details', { ns: 'distribution' })}
+            {t('heading.stock-details')}
           </Typography>
 
           <InfoRow label={t('label.name')} value={item.name} />
           <InfoRow label={t('label.code')} value={item.code} />
-          {item.unitName ? (
-            <InfoRow label={t('label.unit')} value={item.unitName} />
+          {variantsControl ? (
+            <InfoRow
```

Comment:
I was thinking this could be the select drop down like in item catalog, but I think it's ok for now, maybe look at revisiting it in the final touches